test/api/users.coffee

```diff
@@ -0,0 +1,85 @@
+'use strict'
+
+User = require("../../website/src/models/user").model
+app = require("../../website/src/server")
+
+describe "Users", ->
+
+  describe "Deleting Users", ->
+
+    before (done) ->
+      registerNewUser ->
+        done()
+      , true
+
+    it 'Should delete a user', (done) ->
+      async.waterfall [
+        (cb) ->
+          registerManyUsers 1, cb
```

Comment:
See my comment on line 10. That'd basically make this test:
```coffeescript
it 'Should delete a user', (done) ->
  request.del(baseURL + "/user")
    .end (res) ->
        expectCode res, 200
        done()
```

In addition, inside the `.end(res) ->` block, you could do a look up for that user and verify that it 404s.